pkg/teldrvr/newrelicAPM.go

```diff
@@ -170,7 +169,7 @@ func (t *APMTransaction) SegmentEnd(segmentID string) error {
 // Error logs errors in the transaction
 func (t *APMTransaction) Error(_ string, readCloser io.ReadCloser) error {
 	// max bytes available for the error message
-	errMsg := make([]byte, telemetry.ErrorBytesSize)
+	errMsg := make([]byte, telemetry.DebugByteSize)
```

Comment:
Note: DebugBiteSize needs to be declared in telemetry.go in mc-telemetry. The same as ErrorByteSize